dts/bindings/display/backlight.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, Martin Kiepfer
+# SPDX-License-Identifier: Apache-2.0
+
+include: [base.yaml]
+
+properties:
+  init-level:
+    type: int
+    default: 80
```

Comment:
default value must have justification in description https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values